src/backend/distributed/operations/shard_split.c

```diff
@@ -1315,27 +1314,21 @@ NonBlockingShardSplit(SplitOperation splitOperation,
 			sourceShardToCopyNode,
 			workersForPlacementList);
 
-		CreateReplicaIdentities(mapOfDummyShardToPlacement,
-								shardGroupSplitIntervalListList, workersForPlacementList);
-
-		/* 3) Create Publications. */
-		CreateShardSplitPublications(sourceConnection, shardSplitHashMapForPublication);
-
 		/*
-		 * 4) Create template replication Slot. It returns a snapshot. The snapshot remains
-		 * valid till the lifetime of the session that creates it. The connection is closed
-		 * at the end of the workflow.
+		 * 3) Create replica identities on dummy shards. This needs to be done
+		 * before the subscriptions are created. Oth the subscription creation
```

Comment:
Typo

Reply:
FYI, this button is really nice for such purposes: 
![image](https://user-images.githubusercontent.com/1162278/184301810-1a6ff05a-d8c2-45d4-968e-52f5a3d53743.png)


```suggestion
		 * before the subscriptions are created. Otherwise the subscription creation
```